app/controllers/campaign_requests_controller.rb

```diff
@@ -1,7 +1,7 @@
 require 'zendesk/ticket/campaign_request_ticket'
```

Comment:
It looks like the changes to this file could be rebased into the first commit as it's to do with the updated campaign form rather than the live campaign form.